app/services/edition_presenter_factory.rb

```diff
@@ -0,0 +1,16 @@
+module EditionPresenterFactory
```

Comment:
What's the reasoning behind this being a module rather than a class with class methods? I'd not seen `module_function` before, it's a little confusing.